### DISCUSSION TITLE
Elasticsuite Premium Feature Set

### DISCUSSION THREAD

**Discussion by FahrenholzLemundo** [Smile-SA/elasticsuite#3346] [General]
Dear Elasticsuite-Team,

we currently use the elasticsuite free version for our clients magento 2 shop and we are evaluating if it is worth to upgrade to the premium version.
We are using the elasticsuite module for the category filters and the search and it works well.

Now our client is looking for a specific feature. 

Let me describe the current state:
1. The client has a promotions category where all discounted products are shown.
2. The discounted products are also all assigned to another category.
3. The attibutes of the discounted products are not enough to filter for a certain product e.g. Color. 

Therefore the client wants to have a category filter on this promotions category so that you can filter for e.g. Equipment.
We are not sure if we should build it by our own or if your virtual attributes might already help since the categories could change from time to time. (https://elastic-suite.github.io/documentation/docs/ElasticSuite/Virtual%20Attributes%20Rules/Products/Rules%20creation.html)

Is this achievable without further (or at least little) effort by the tools like virtual attributes provided by the premium version?
Maybe there is even an easy open source solution for it?

Thanks in advance!
Kindest Regards from Lemundo

**Comment by Bashev** [comment]
Are you trying to enable "Generate Virtual Category Root Subtree" in Promotions category? 
If i understand properly this is what you are looking for.

**Comment by rbayet** [comment]
Hello @FahrenholzLemundo,

From what I understand, I second Bashev's opinion.

The Premium Virtual Attributes module typical use is more to restrict / clean / rationalize attribute data.
For instance, let's imagine you lack a PIM and receive product information from manufacturers with an color names like "Reddish", "Winter blue sky", "Energized yellow", "Auburn", "Azur".
You might want to provide a "simple color"  navigation filter attribute with "red", "blue", "yellow", etc...
That module will help you create matching rules to associate a selection of complex color attribute options to a simple color attribute option.

Regards,

**Comment by FahrenholzLemundo** [comment]
Wow, that is exactly what we want and it is even in the free version! 🤩
Didn't know that is possible with the virtual category feature. Thank you guys so much! 💪
